web_src/js/features/common-global.js

```diff
@@ -169,6 +170,8 @@ export function initGlobalDropzone() {
             let fileMarkdown = `[${file.name}](/attachments/${file.uuid})`;
             if (file.type.startsWith('image/')) {
               fileMarkdown = `!${fileMarkdown}`;
+            } else if (file.type.startsWith('video/')) {
+              fileMarkdown = `<video src="/attachments/${file.uuid}" title="${htmlEscape(file.name)}" controls></video>`;
```

Comment:
```suggestion
              fileMarkdown = htmlEscape`<video src="/attachments/${file.uuid}" title="${file.name}" controls></video>`;
```

May as well escape both.

Reply:
Not sure I understand, we don't want to escape e.g. the `<` in `<video`?

Reply:
It will only escape the interpolated values, e.g.:

```js
> htmlEscape`<video src="/attachments/${"&<"}" title="${">&"}" controls></video>`;
'<video src="/attachments/&amp;&lt;" title="&gt;&amp;" controls></video>'
```

Reply:
If that is too much magic to you, I'm also happy to accept two `htmlEscape` calls 😉.

Reply:
I don't think the UUID needs escaping? But if it does, we'd need to use something like `encodeURIComponent` instead?

Reply:
Ah you are right, `encodeURIComponent` should be used and its output does not need HTML escaping as it will percent-escape which is "safe". I guess we can keep it as-is because if UUID would contain slashes, it would incorrectly escape them, and this value is controlled by us anyways.